spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkWriteConf.java

```diff
@@ -75,6 +76,10 @@ public class SparkWriteConf {
 
   private static final Logger LOG = LoggerFactory.getLogger(SparkWriteConf.class);
 
+  private static final long DATA_FILE_SIZE = 128 * 1024 * 1024; // 128 MB
```

Comment:
These are some default values we will try to achieve. They should be safe without affecting the parallelism too much.